### PR TITLE
Add i18n support for official website

### LINKED ISSUE #1483: Add i18n support for official website
**What problem or use case are you trying to solve?**

https://github.com/OpenDevin/OpenDevin/pull/1372 refactored the docs and created a [website](https://opendevin.github.io/OpenDevin/) for hosting docs, which is great! But that PR also removes [README-zh](https://github.com/OpenDevin/OpenDevin/blob/46bd83678a09c9f20ab53b30b7f2278932ace11b/docs/README-zh.md) file, a Chinese translation of official doc. Instead of getting the chinese translation of README back, I think we probably should leverage the chance and introduce i18n support for the official website.

**Describe the UX of the solution you'd like**

Add i18n support for the official website and/or doc. I guess when basic support is added, we can use OpenDevin to dogfood and finish the translations.

### PR DESCRIPTION
**Pull request by Umpire2018**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**

Fix #1483 

**Give a brief summary of what the PR does, explaining any non-trivial design decisions**

- zh-Hans
 
image

image

- fr

image

image

**Other references**

1. Used `gpt-4o` to finish the translation work.
2. Uploading the `i18n` folder to Git may not be the best practice. `Docusaurus` are using [Crowdin](https://docusaurus.io/docs/i18n/crowdin) to build a workflow of translation. Maybe starte with setting a specific API key for translation, we can integrate with Crowdin and streamline the localization process. 
3. `i18n` is an ongoing process that requires patience and dedication. I welcome more people to join me in this effort to make our documentation accessible to a wider audience.

### CODE REVIEW

File: docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+OpenDevin 使用 Docker 容器来安全地完成工作，而不会破坏您的机器。
+
+### 解决方法
+
+* 运行 `docker ps` 以确保 Docker 正在运行
+* 确保您不需要使用 `sudo` 运行 Docker [请参见此处](https://www.baeldung.com/linux/docker-run-without-sudo)
+* 如果您使用的是 Mac，请检查[权限要求](https://docs.docker.com/desktop/mac/permission-requirements/) ，特别是考虑在 Docker Desktop 的 `Settings > Advanced` 下启用 `Allow the default Docker socket to be used`。
+* 另外，升级您的 Docker 到最新版本，选择 `Check for Updates`
+
+## 无法连接到 SSH 盒子
```
Comment (yufansong) [inline docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md:44]:
```suggestion
## 无法连接到 DockerSSHBox
```

File: docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+[GitHub 问题](https://github.com/OpenDevin/OpenDevin/issues/1156)
+
+### 症状
+
+```python
+self.shell = DockerSSHBox(
+...
+pexpect.pxssh.ExceptionPxssh: 无法建立与主机的连接
```
Comment (yufansong) [inline docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md:53]:
```suggestion
pexpect.pxssh.ExceptionPxssh: Could not establish connection to host
```

File: docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+这通常发生在本地 LLM 设置中，当 OpenDevin 无法连接到 LLM 服务器时。请参阅我们的 [本地 LLM 指南](llms/localLLMs) 以获取更多信息。
+
+### 解决方法
+
+* 检查您的 `config.toml` 文件中 "llm" 部分的 `base_url` 是否正确（如果存在）
+* 检查 Ollama（或您使用的其他 LLM）是否正常运行
+* 确保在 Docker 中运行时使用 `--add-host host.docker.internal:host-gateway`
+
+## `404 资源未找到`
```
Comment (yufansong) [inline docs/i18n/zh-Hans/docusaurus-plugin-content-docs/current/usage/troubleshooting/troubleshooting.md:93]:
```suggestion
## `404 Resource not found 资源未找到`
```

**Review by yufansong — COMMENTED** [review]
Chinese doc LGTM. But I don't know the french or the doc code. Hope others can help to check.

**Comment by neubig** [comment]
Thanks @Umpire2018 , this looks nice! But tests are failing, so we'll need to fix those.

Uploading the i18n folder to git might not be best practice, but honestly it seems good enough for now.

**Comment by Umpire2018** [comment]
I have made attempts to eliminate the following issue:

```sh
[WARNING] Docusaurus found broken anchors!

Please check the pages of your site in the list below, and make sure you don't reference any anchor that does not exist.
Note: it's possible to ignore broken anchors with the 'onBrokenAnchors' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken anchors found:

Broken anchor on source page path = /OpenDevin/fr/modules/usage/architecture:
-> linking to #backend-architecture-fr (resolved as: /OpenDevin/fr/modules/usage/architecture#backend-architecture-fr)
```

However, I was not successful. Nevertheless, this warning does not affect the build.

image

**Comment by enyst** [comment]
This is awesome! French works too, at least at a cursory reading at 1:00 a.m. As you say, i18n is an on-going process, it's great to get it started. ❤️

**Review by neubig — COMMENTED** [review]
Hi @Umpire2018 , really sorry this took so long to merge in and then I made some changes to the doc that broke this PR :/ Would you be able to update it? Thank you!

**Comment by yufansong** [comment]
Hey @Umpire2018, it seems all check pass. Can you solve the conflict before we merge this PR?

**Comment by Umpire2018** [comment]
@neubig @yufansong Hi, the conflict is solved and ready to merge.

**Review by yufansong — APPROVED** [review]

**Pull request merged**